integrations/datadog/.port/resources/blueprints.json

```diff
@@ -194,5 +202,39 @@
         "many": true
       }
     }
+  },
+  {
+    "identifier": "datadogSloHistory",
+    "description": "This blueprint represents a datadog SLO history",
+    "title": "Datadog SLO History",
+    "icon": "Datadog",
+    "schema": {
+      "properties": {
+        "timeframe": {
+          "title": "Timeframe",
+          "type": "string"
+        },
+        "thresholds": {
+          "title": "Thresholds",
+          "type": "object"
+        },
+        "sliValue": {
+          "title": "SLI Value",
+          "type": "string"
+        }
+      },
+      "required": []
+    },
+    "mirrorProperties": {},
+    "calculationProperties": {},
+    "relations": {
+      "monitors": {
+        "target": "datadogSlo",
+        "title": "SLO",
+        "description": "The SLO to which this history belongs to",
+        "required": false,
+        "many": true
+      }
```

Comment:
looks from the mapping that its one to one relation and should be required I guess